### PR TITLE
dotCMS/core#24161 fix DotFavoritePage-empty-state-styles

### LINKED ISSUE #24161: UI - Favorite Page portlet: Add empty state design on contentlets table
### Parent Issue
#22343 

image

design mock - WIP

### PR DESCRIPTION
**Pull request by alfredo-dotcms**
### Proposed Changes
image

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Comment by alfredo-dotcms** [comment]
fix:
image

**Review by zJaaal — COMMENTED** [review]
see my comments

File: core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html [hunk 1]
```diff
@@ -59,13 +59,15 @@
 
     <ng-template #emptyState>
         <div class="dot-pages-empty__container">
-            <dot-icon name="star_outline" size="18"></dot-icon>
+            <dot-icon name="star_outline" size="32"></dot-icon>
```
[the remark this answers is not in the document]
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html:62]: @fmontes thoughts?

[the remark this answers is not in the document]
Comment (fmontes) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html:62]: True that!

[the remark this answers is not in the document]
Comment (alfredo-dotcms) [reply core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.html]: @zJaaal changed!

**Review by zJaaal — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Pull request merged**